src/sentry/receivers/outbox/region.py

```diff
@@ -94,15 +96,22 @@ def process_organization_member_updates(
 
     rpc_org_member_update = RpcOrganizationMemberMappingUpdate.from_orm(org_member)
 
-    organizationmember_mapping_service.update_with_organization_member(
-        organizationmember_id=org_member.id,
-        organization_id=shard_identifier,
-        rpc_update_org_member=rpc_org_member_update,
-    )
+    if _was_monolith():
```

Comment:
How do non-US regions update membership mappings?

Reply:
Oh, right, what I need to do is simply remove the organizationmember_id=org_member.id, parameter, not the entire update, and write a test for that. Good catch.

Reply:
Oh, right, what I need to do is simply remove the organizationmember_id=org_member.id, parameter, not the entire update, and write a test for that. Good catch.

Reply:
Took a different approach -- take another pass?